BD-2027: Add /users/track with email address EA (#5185)
braze-inc/braze-docs: _docs/_api/endpoints/user_data/post_user_track.md

}
```

## Example request for updating a user profile by email address

{% alert important %}

> Updating a user profile by email address with this endpoint is currently in early access. Contact your Braze customer success manager if you're interested in participating in the early access.

Author: ```suggestion
Updating a user profile by email address with this endpoint is currently in early access. Contact your Braze account manager if you're interested in participating in the early access.
```

                "alias_name": "device123",
                "alias_label": "my_device_identifier"
            },
            "app_id": "your_app_identifier",
            "name": "rented_movie",
            "time": "2013-07-16T19:20:50+01:00"
        }
    ],
    "purchases": [
        {
            "email": "test@braze.com",
            "app_id": "your_app_identifier",
            "product_id": "product_name",
            "currency": "USD",
            "price": 12.12,
            "quantity": 6,
            "time": "2017-05-12T18:47:12Z",
            "properties": {
                "color": "red",
                "monogram": "ABC",
                "checkout_duration": 180,
                "size": "Large",
                "brand": "Backpack Locker"
            }
        }
    ]
}`
```

### Frequently asked questions

#### What happens when multiple profiles with the same email address are found?
If the `external_id` exists, the most recently updated profile with an external ID will be prioritized for updates. If the `external_id` doesn't exist, the most recently updated profile will be prioritized for updates.

#### What happens if no profile with the email address currently exists?

> A new profile will be created and an email-only user will be created.  There will be no alias created. The email field will be set to test@braze.com as noted in the example request for updating a user profile by email address.

Author: ```suggestion
A new profile will be created and an email-only user will be created. An alias will not be created. The email field will be set to test@braze.com, as noted in the example request for updating a user profile by email address.
```
